client/my-sites/plans/jetpack-plans/product-store/items-list/all-items.tsx

```diff
@@ -95,14 +96,24 @@ export const AllItems: React.FC< AllItemsProps > = ( {
 						<li key={ item.productSlug }>
 							<SimpleItemCard
 								ctaAsPrimary={ ctaAsPrimary }
-								ctaHref={ getCheckoutURL( item ) }
+								ctaHref={
+									item.productSlug === PRODUCT_JETPACK_SOCIAL_BASIC &&
+									config.isEnabled( 'jetpack-social/advanced-plan' )
+										? '#'
+										: getCheckoutURL( item )
+								}
```

Comment:
I think it would be good to move this logic to a separate `const` declaration above the return statement, with a comment explaining what it's doing.